androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/NextCircleButton.kt

```diff
@@ -0,0 +1,39 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+
+@Composable
+fun NextCircleButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
```

Comment:
Any reason why we don't use `FloatingActionButton`?

Reply:
Came from the other branch after rebasing. I'll remove it.